TrafficCapture/dockerSolution/src/main/docker/openSearchBenchmark/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:focal
```

Comment:
why this particular version?

Reply:
https://opensearch.atlassian.net/browse/MIGRATIONS-1138